recruitment/positions/android_chapter_lead.md

```diff
@@ -0,0 +1,86 @@
+<img src="../../logo.png">
+</p>
+
+Android Chapter Lead
+==================================
+
+Babylon aims to hire only the most qualified people available for the 
+job at any given time. That doesn't mean that a candidate must have 
+several academic degrees or have years and years of experience. 
+We value potential equally as much as knowledge and experience. 
+Above all, we value one's ability to think independently and to engage.
+
+Additionally, we want to hire people who are reasonable, professional, 
+and fun to work with. After all, they are people we will spend a great 
+deal of time interacting with on a daily basis.
+
+We are continually improving how we work remotely as a company and aim 
+to be as flexible as possible by equally accepting applications from 
+candidates who want to work on-site at our office in London, or who 
+ant to work remotely.
+
+At Babylon we also believe in/champion diversity and welcome everyone 
+from everywhere. Currently in the Android team you'll find people from 
+Greece, Poland, UK, India, Spain, Portugal, France, Lithuania, Sudan, 
+Italy and Hungary, can you add to it?
+
+#### You'd need to
+
+- Ensure high-quality work is delivered by the engineers in the chapter
+- Line manage up to 6 engineers. Mentoring, supporting and conducting 
+  regular 1:1s
+- Code. Your time will be split about 50/50 between managing & individual 
+  contribution
+- Contribute to planning and estimation in your squad
+- Help establishing best practices and drive positive change
+
+####  You've likely got:
+
+- 5+ years of software development experience designing and building 
+  best-in-class native mobile experiences
+- Excellent knowledge of Kotlin
+- Understanding of software engineering and project management 
+  techniques, including agile delivery and estimating
+- Meaningful experience with an FRP framework (e.g. RxJava)
+- The ability to make difficult decisions, and hold a high bar for the 
+  quality of the engineers who you lead
+- Excellent oral and written communication skills. Explaining technical 
+  issues in a clear way to business and other stakeholders
+
+#### And we'd be super impressed if:
+
+- You contribute to the Android community, through things like projects 
+  on GitHub, blog posts or giving talks
+
+#### Want to join?
+  
+If you like what you read and want to join the team and help us 
+build the future of healthcare, you can 
+[apply here](https://jobs.lever.co/babylonhealth/4e586b2b-05aa-4cd0-8353-553c288df3f0)
+
+#### Interview Structure
+
+The interview process for the Android Chapter Lead role generally 
+consists in the stages/sessions below. 1 and 2 always happen
+off-site and from 3-7 depending if you're based around London or not,
+some of the sessions might happen remotely as well but they usually
+happen in a half day on-site visit. 
+
+Check our [interview stages page](../interview_stages) for a brief 
+description of each of the stages/sessions.
+ 
+1. Initial phone call
```

Comment:
you can put links for all of these I guess.

Reply:
I would have to make the titles simpler in the other page, as they are right now it's really hard. Let me think about on how to make it better

Reply:
cool.

Reply:
moved the duration and people involved out of the titles and added the links here and in the other role